optuna/integration/tensorflow.py

```diff
@@ -35,13 +35,9 @@ class TensorFlowPruningHook(SessionRunHook):
             An evaluation metric for pruning, e.g., ``accuracy`` and ``loss``.
         run_every_steps:
            An interval to watch the summary file.
-        is_higher_better:
-           Please do not use this argument because this class refers to
-           :class:`~optuna.structs.StudyDirection` to check whether the current study is
-           ``minimize`` or ``maximize``.
     """
 
-    def __init__(self, trial, estimator, metric, run_every_steps, is_higher_better=None):
+    def __init__(self, trial, estimator, metric, run_every_steps):
         # type: (optuna.trial.Trial, tf.estimator.Estimator, str, int, Optional[bool]) -> None
```

Comment:
Please update the type hinting.
```suggestion
        # type: (optuna.trial.Trial, tf.estimator.Estimator, str, int) -> None
```

Reply:
Thanks!
I applied your suggested change.